hawkbit-autoconfigure/src/main/java/org/eclipse/hawkbit/autoconfigure/security/SecurityAutoConfiguration.java

```diff
@@ -82,4 +88,33 @@ public SecurityTokenGenerator securityTokenGenerator() {
         return new SecurityTokenGenerator();
     }
 
+    /**
+     * @return {@link LogoutHandler} bean
+     */
+    @Bean
+    @ConditionalOnMissingBean
+    public AuthenticationSuccessHandler authenticationSuccessHandler() {
+        return new SimpleUrlAuthenticationSuccessHandler();
+    }
+
+    /**
+     * @return {@link LogoutHandler} bean
+     */
+    @Bean
+    @ConditionalOnMissingBean
+    public LogoutHandler logoutHandler() {
+        return new SecurityContextLogoutHandler();
+    }
+
+    /**
+     * @return {@link LogoutSuccessHandler} bean
+     */
+    @Bean
+    @ConditionalOnMissingBean
+    public LogoutSuccessHandler logoutSuccessHandler() {
+        final SimpleUrlLogoutSuccessHandler simpleUrlLogoutSuccessHandler = new SimpleUrlLogoutSuccessHandler();
+        simpleUrlLogoutSuccessHandler.setTargetUrlParameter("login");
```

Comment:
just a suggestion to use a constant here for the string value `login`

Reply:
The String is just used once here, so it would be just an extra line of code. I just let it the way it is :)